src/java/voldemort/store/StoreDefinition.java

```diff
@@ -573,6 +573,10 @@ private void addToDiff(String propertyName, long thisValue, long otherValue, Str
     private void addToDiff(String propertyName, int thisValue, int otherValue, StringBuilder sb, String thisName, String otherName) {
         addToDiff(propertyName, Integer.toString(thisValue), Integer.toString(otherValue), sb, thisName, otherName);
     }
+    
+    private void addToDiff(String propertyName, Object thisValue, Object otherValue, StringBuilder sb, String thisName, String otherName) {
```

Comment:
Would you mind using `String.valueOf` at the call-site rather than changing the signature? It seems cleaner to have a strong type in the function signature. Not a big deal either way. Just a slight preference.